src/renault_api/kamereon/models.py

```diff
@@ -198,11 +198,23 @@ class EndpointDefinition:
     "XBG1VE": {  # DACIA SPRING
         "actions/charge-start": _KCM_ENDPOINTS["actions/charge-start"],
         "actions/charge-stop": _KCM_ENDPOINTS["actions/charge-stop"],
+        "alerts": None,  # Reason: "err.func.wired.not-found"
         "battery-status": _DEFAULT_ENDPOINTS["battery-status"],
-        "charge-mode": None,
+        "charge-history": None, # Reason: "err.func.wired.not-found"
```

Comment:
Pre-commit needs two spaces before all the `#`
```suggestion
        "charge-history": None,  # Reason: "err.func.wired.not-found"
```